### PR TITLE
coalesce.py should use $ARTIFACTS if it exists.

### PR DESCRIPTION
**Pull request by Katharine**
Path construction is bad.

/kind cleanup
/cc @BenTheElder

### CODE REVIEW

**Comment by liztio** [comment]
:+1:

File: hack/coalesce.py [hunk 1]
```diff
@@ -77,8 +77,8 @@ def main():
     root.set('time', '0')
     for package in sorted(test_packages('bazel-testlogs')):
         root.append(result(package))
-    artifacts_dir = os.path.join(os.environ.get('WORKSPACE', os.getcwd()),
-                                 '_artifacts')
+    artifacts_dir = os.environ.get('ARTIFACTS',
```
Comment (fejta) [inline hack/coalesce.py]: Break the line before `'ARTIFACTS'`

**Comment by Katharine** [comment]
I ran pylint locally so hopefully I got it right this time.

**Comment by fejta** [comment]
`pylint` is a fickle beast :)

**Comment by BenTheElder** [comment]
Thanks!

**Pull request merged**